tests/typ/compiler/ops.typ

```diff
@@ -301,3 +301,41 @@
 #let inc2 = inc.with(y: 2)
 #test(inc2(2), 4)
 #test(inc2(2, y: 4), 6)
+
+---
+// Test chained comparison operations
```

Comment:
Could you also add

```typ
#test((1 < 2) == (2 < 1), false)
```

I think it should work

Reply:
But that's not really in the scope of this PR tho, since it is valid syntax already before hand